app/lib/user-utils.js

```diff
@@ -140,6 +140,19 @@ function levelInExam (slug) {
   })
 }
 
+function levelNumberInExam (slug) {
+  const me = window.me
+  const exam = localStorage.load(`exam-${me.id}`, true)
+  if (!exam) { return -1 }
+  const levels = []
+  exam.problems.forEach((course) => {
```

Comment:
can this logic and level number logic on ExamPage use the same function for uniformity and DRYing the logic?